virttest/rss_client.py

```diff
@@ -199,7 +199,7 @@ def _receive_packet(self, timeout=60):
     def _send_file_chunks(self, filename, timeout=60):
         if self._log_func:
             self._log_func("Sending file %s" % filename)
-        f = open(filename, "rb")
+        f = open(filename, "r")
```

Comment:
This may deal with binary file, so suggest keep code as before.

Reply:
Ok, will change back to rb